engines/dfc_provider/config/routes.rb

```diff
@@ -2,12 +2,14 @@
 
 DfcProvider::Engine.routes.draw do
   namespace :api do
-    scope :dfc_provider, as: :dfc_provider, path: '/dfc_provider' do
-      resources :enterprises, only: [:show] do
-        resources :catalog_items, only: [:index, :show]
-        resources :supplied_products, only: [:show]
+    scope '/v0' do
```

Comment:
I think we should handle this a bit differently. The `/api/v0` namespace is for internal/deprecated endpoints that are currently used by the main app. We'll be replacing them with public endpoints under `/api/v1` and deleting the `/v0` controllers as we update the frontend (and stop depending on those routes).

I think we should not move the DFC routes under that `v0` namespace. We added a redirect for the routes change [here](https://github.com/openfoodfoundation/openfoodnetwork/blob/bd2484c3e0bed077b5bc7cf620203523788d5f2a/config/routes/api.rb#L82), if it's interfering with the DFC routes we should update it to not do that :+1: